test-configs.yaml

```diff
@@ -1328,22 +1334,32 @@ test_configs:
   - device_type: qcom-qdf2400
     test_plans: [boot]
 
-  - device_type: qemu_arm
+  - device_type: qemu-arm-virt-gicv2
     test_plans:
       - boot_qemu
       - simple_qemu
-      - uefi-arm-gicv2
-      - uefi-arm-gicv3
+      - uefi-arm
       - v4l2_compliance_qemu_vivid
```

Comment:
No need to run v4l2-compliance-vivid twice on arm and arm64, so probably only with `-gicv3` in both cases.